lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_beta.rb

```diff
@@ -23,14 +34,21 @@ def self.run(params)
       #####################################################
 
       def self.description
-        'Bumps the version of the app for a new beta'
+        'Bumps the version of the app for a new beta. Depends on a gradle task to update the keys in a version.properties file.'
```

Comment:
```suggestion
        'Bumps the version of the app for a new beta. Requires the `updateVersionProperties` gradle task to update the keys if you are using a `version.properties` file.'
```

(And same suggestion (search/replace all) on all the places and other actions where you mentioned this)

Reply:
That's true! Agree with being a bit more verbose here